fake.py

```diff
@@ -427,6 +438,15 @@ def getNextOddNumber(number):
         return number + 1
     return number
 
+def getPercentage(number):
+    return min(max(int(number), 0), 100)
```

Comment:
Surely we can just use ``round`` here?

Reply:
Here I'm letting the user input a number between 0 and 100 and limiting it to 0 or 100 if it's lower or higher.

Reply:
I suppose it is better to sanitise user input like what you did here.